### ISSUE TITLE
Missing ```package:angel/angel.dart```

### ISSUE THREAD

**Issue by eyasuyuki** [dukefirehawk/angel#1]
I try to Angel3 like a README.md.

```
pub global activate angel3_cli
angel3 init hello
cd hello
dart --observe bin/dev.dart
```

Then, I got some errors. I think ```package:angel/angel.dart``` is missing.

```
Observatory listening on http://127.0.0.1:8181/Juqp405NSGg=/
Error: Couldn't resolve the package 'angel' in 'package:angel/angel.dart'.
bin/dev.dart:7:8: Error: Not found: 'package:angel/angel.dart'
import 'package:angel/angel.dart';
       ^
bin/dev.dart:18:25: Error: Getter not found: 'configureServer'.
    await app.configure(configureServer);
                        ^^^^^^^^^^^^^^^
```

My Dart version:

```
Dart SDK version: 2.13.4 (stable) (Wed Jun 23 13:08:41 2021 +0200) on "macos_x64"
```

**Comment by dukefirehawk** [comment]
Angel3_cli is still work in progress at the moment. Internally it is using the boilerplate code from
1. https://github.com/dukefirehawk/boilerplates/tree/angel3-basic
2. https://github.com/dukefirehawk/boilerplates/tree/angel3-orm
3. https://github.com/dukefirehawk/boilerplates/tree/angel3-graphql

Just do a git pull from one of the above to get started.

**Comment by dukefirehawk** [comment]
Resolved in version 3.0.0.
Run `dart pub global activate angel3_cli` to upgrade it.
